stormlog/infer/samplers.py

```
@@ -0,0 +1,119 @@
+        self.device_id = device_id
+
+    def sample(self, *, session_id: str) -> InferenceSystemSample | None:
+        result = subprocess.run(
+            [
+                "nvidia-smi",
+                (
+                    "--query-gpu=index,name,memory.total,memory.used,"
+                    "memory.free,utilization.gpu"
+                ),
+                "--format=csv,noheader,nounits",
+            ],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)